server/db/_db.js

```diff
@@ -7,6 +7,6 @@ global._dbPath = path.join(__dirname, "_db.js");
 var env = require(path.join(__dirname, '../env'));
```

Comment:
And now we are adding models into this PR? That should be its own PR. Keep things segregated to keep them clean.